docs/install/jetstream.md

```diff
@@ -1,152 +1,101 @@
 
-    ```{image} ../images/providers/jetstream/launch-instance-dialog.png
-    :alt: Launch an Instance / Basic Options dialog box
-    ```
+Encrypted (HTTPS) access is strongly recommended before inviting users.
 
-    1. Give your server a descriptive **Instance Name**.
+See the full guide: [Enable HTTPS](/howto/admin/https). Below is a quick recipe for using the default Jetstream-provided hostname.
 
-    2. Select an appropriate **Instance Size**. We suggest m1.medium or larger.
-       Make sure your instance has at least **1GB** of RAM.
+1. In the terminal inside the instance, configure Let's Encrypt (replace with a real email you control):
+   ```bash
+   sudo tljh-config set https.enabled true
+   sudo tljh-config set https.letsencrypt.email you@example.com
+   sudo tljh-config add-item https.letsencrypt.domains yourinstancename.xxx0000000.projects.jetstream-cloud.org
+   sudo tljh-config reload proxy
+   ```
+2. Wait ~30–60 seconds, then reload the site using https://. If certificate issuance fails, check the logs:
+   ```bash
+   sudo journalctl -u traefik --since "10 minutes ago" | grep -i acme
+   ```
 
-       Check out our guide on How To [](/howto/admin/resource-estimation) to help pick
-       how much Memory, CPU & disk space your server needs.
+Tips:
 
-    3. If you have multiple allocations, make sure you are 'charging' this server
-       to the correct allocation.
+- Make sure ports 80 and 443 are open in your Jetstream security group (they are open by default for new projects; adjust only if you customized network policies).
+- If you later attach a custom domain, add it with another `add-item` command and reload the proxy again.
 
-6.  Click the **Advanced Options** link in the bottom left of the popup. This
-    lets us configure what the server should do when it starts up. We will use
-    this to install The Littlest JupyterHub.
+## Step 3: Customize your JupyterHub deployment
 
-    A dialog titled **Launch an Instance / Advanced Options** should pop up.
+1. Now log in with the `<admin-user-name>` at https://yourinstancename.xxx000000.projects.jetstream-cloud.org. Since this is the first login, you'll be prompted to set a password. Choose a strong password and store it safely. This password is now the credential for that admin user.
 
-    ```{image} ../images/providers/jetstream/add-deployment-script-dialog.png
-    :alt: Dialog box allowing you to add a new script.
-    ```
+Next common tasks:
 
-7.  Click the **Create New Script** button. This will open up another dialog
-    box!
+- Add additional users or admins: [User management](/howto/admin/add-users)
+- Install Python / conda packages for all users: [Customize user environment](/howto/user-env)
+- Install optional system packages: `sudo apt install <package>` (restart user servers afterwards)
+- Pre-install JupyterLab extensions: see [Enable extensions](/howto/admin/enable-extensions)
+- Update TLJH itself: [Upgrading TLJH](/topic/installer-upgrade-actions)
```

Comment:
I suspect we only need to specify the link URLs, the titles should be populated automatically.

```suggestion
- [](howto-admin-admin-users)
- [](howto-user-env-user-environment-apt)
- [](howto-admin-enable-extensions)
- [][topic-installer-upgrade-actions)
```